### PR TITLE
Deprecating the CLI

### PR DESCRIPTION
**Pull request by bvallelunga**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# DEPRECATED
+This library has been deprecated. The new Doppler CLI binary is exceptionally light and fast.
```
Comment (Piccirello) [inline README.md]: The new Doppler CLI is exceptionally light and fast, and is provided as a binary to support environments without `node`.

**Comment by bvallelunga** [comment]
Changed the `update` command.

File: controllers/update.js [hunk 2]
```diff
@@ -1,5 +1,18 @@
...
-  program.utils.runCommand("npm install --global @dopplerhq/cli")
+const open = require("open")
+const chalk = require("chalk")
+const enquirer = require("enquirer")
+
+async function task_runner(program, argument, options) {
+  const prompt = new enquirer.Confirm({
+    type: 'confirm',
+    name: 'open',
+    message: chalk.yellow("Open browser to the upgrade."),
```
Comment (bvallelunga) [inline controllers/update.js]: Screen Shot 2019-12-11 at 4 30 21 PM

File: controllers/update.js [hunk 3]
```diff
@@ -1,5 +1,18 @@
-function task_runner(program, argument, options) {  
-  program.utils.runCommand("npm install --global @dopplerhq/cli")
+const open = require("open")
+const chalk = require("chalk")
+const enquirer = require("enquirer")
+
+async function task_runner(program, argument, options) {
+  const prompt = new enquirer.Confirm({
+    type: 'confirm',
```
Comment (bvallelunga) [inline controllers/update.js]: Codacy Issue found: [Strings must use doublequote. (quotes)](https://app.codacy.com/app/Doppler/node-cli/pullRequest?prid=4662036)

File: controllers/update.js [hunk 4]
```diff
@@ -1,5 +1,18 @@
...
+    name: 'open',
```
Comment (bvallelunga) [inline controllers/update.js]: Codacy Issue found: [Strings must use doublequote. (quotes)](https://app.codacy.com/app/Doppler/node-cli/pullRequest?prid=4662036)

**Pull request merged**